sentence_test.go

```diff
@@ -76,6 +82,8 @@ func TestSentences(t *testing.T) {
 				assert.NoError(t, err)
 				assert.Equal(t, tt.sent, sent)
 				assert.Equal(t, tt.sent.Raw, sent.String())
+				assert.Equal(t, tt.datatype, sent.DataType())
+				assert.Equal(t, tt.talkerid, sent.TalkerID())
```

Comment:
can you please assert the prefix as well?

Reply:
Will do.
I see tabs are used in the README file. Should we replace with three spaces or keep using tabs?

Reply:
AFAIK, tabs is how `go fmt` indents lines.

Reply:
I've pushed now. Feel free to resolve this conversation if the push matched your request :-).

Reply:
LGTM